bin/checkver.ps1

```diff
@@ -65,7 +65,7 @@ $queue | % {
 
     if ($json.checkver -eq "github") {
         if (!$json.homepage.StartsWith("https://github.com/")) {
-            write-host "ERROR: $name checkver expects the homepage to be a github repository" -f DarkYellow
+            error "$name checkver expects the homepage to be a github repository"
```

Comment:
Even though the original was darkyellow text, changed to `error` instead of `warn`, since it seemed like an error condition.